tasks/scripts.mjs

```diff
@@ -38,10 +41,22 @@ export async function compileJavaScript ([modulePath, { srcPath, destPath, fileP
   const moduleSrcPath = join(srcPath, modulePath)
   const moduleDestPath = join(destPath, filePath ? filePath(parse(modulePath)) : modulePath)
 
+  // Rollup plugins
+  const plugins = []
+
+  if (!isDev) {
+    // Add GOV.UK Frontend release version
+    // @ts-expect-error "This expression is not callable" due to incorrect types
```

Comment:
By using [`"moduleResolution": "Node"`](https://www.typescriptlang.org/docs/handbook/module-resolution.html) we can resolve this in another PR

Although it still works, not all packages are typed to say "I work as ESM-only"